boot/espressif/hal/include/mcuboot_config/mcuboot_config.h

```diff
@@ -45,23 +45,43 @@
  * the default upgrade mode.
  */
 
-/* Uncomment to enable the overwrite-only code path. */
-/* #define MCUBOOT_OVERWRITE_ONLY */
+/* Define to enable the swap-using-move code path. */
+#if defined(CONFIG_BOOT_SWAP_USING_MOVE)
+#define MCUBOOT_SWAP_USING_MOVE 1
+#endif
```

Comment:
Since this is the Espressif Port `mcuboot_config.h` file, I suggest to keep the naming pattern, like `CONFIG_ESP_BOOT_SWAP_USING_MOVE`, same for the others below.
```suggestion
/* Define to enable the swap-using-move code path. */
#if defined(CONFIG_ESP_BOOT_SWAP_USING_MOVE)
#define MCUBOOT_SWAP_USING_MOVE 1
#endif
```

Reply:
Ok, makes sense. Done.